# Service to Stop velocity tracking
---
int64 ret
string msg
